src/main/java/seedu/address/storage/JsonSerializableTutorsPet.java

```diff
@@ -20,15 +21,19 @@
 class JsonSerializableTutorsPet {
 
     public static final String MESSAGE_DUPLICATE_STUDENT = "Students list contains duplicate student(s).";
+    public static final String MESSAGE_DUPLICATE_MODULE_CLASS = "Class list contains duplicate student(s).";
```

Comment:
I think you missed out on this:
```suggestion
    public static final String MESSAGE_DUPLICATE_MODULE_CLASS = "Class list contains duplicate class(es).";
```

Reply:
Resolved by adopting suggested change